### ISSUE TITLE
Diff should show class comments.

### ISSUE THREAD

**Issue by npasserini** [npasserini/iceberg#209]

**Comment by cdlm** [comment]
Yup. I just wrote a bunch of class comments and the commit window only shows the class definitions…

**Comment by estebanlm** [comment]
but... this is not so easy :(
we use MC metamodel and it does not incorpores it when creating the diff.  
I would say we also need a diff for all packages and not package-to-packages as we have now :)

**Comment by dalehenrich** [comment]
absolutely, you need to look into git-based diffs - which provide an overview of ALL changes made during the commit --- then you can identify renames of methods, etc. Using MC for diffs loses all kinds of information that is there in the git diff  ... folks checkin updates to README files and these changes need to be shown as part of the diff ... it isn't quite as hard as you may think ... I've had Smalltalk code for several years now that parses git diff information ... the challenge is coming up with a pretty interface --- and I agree that is not easy:)
